dronegen/tag.go

```diff
@@ -99,6 +99,21 @@ func tagPipelines() []pipeline {
 		},
 	}))
 
+	ps = append(ps, ghaBuildPipeline(ghaBuildType{
+		buildType:    buildType{os: "linux", fips: false},
+		trigger:      triggerTag,
+		pipelineName: "build-teleport-spacelift-runner-oci-images",
+		workflows: []ghaWorkflow{
+			{
+				name:              "release-teleport-spacelift-runner-oci.yml",
+				srcRefVar:         "DRONE_TAG",
+				ref:               "${DRONE_TAG}",
+				timeout:           150 * time.Minute,
+				shouldTagWorkflow: true,
```

Comment:
Should this be sending notifications to slack when this fails? If so, you can add `slackOnError: true,` to the `ghaWorkflow`. It doesn't matter all that much - I'm in the process of setting up GHA to send the notifications to slack when workflows fail, but it's still an option here.

Same applies to promotion.

Reply:
I copied the behaviour of `kube-agent-updater`'s image build here. More than happy to add this in but as I've already tested this, perhaps easier to leave it out and wait for your new behaviour.

Reply:
That's fine. With any luck, I'll be deleting this by the end of the week ;-) - It will get moved to a top-level github actions workflow; no more Drone.

Reply:
Wahoo! Great news and great work - thank you!